public/actions/TagActions/getTag.js

```diff
@@ -32,7 +37,22 @@ export function getTag(id) {
     return dispatch => {
         dispatch(requestTagGet());
         return tagManagerApi.getTag(id)
-            .then(res => dispatch(recieveTagGet(res)))
+            .then(res => {
+
+              const store = getStore();
+              var permitted = store.getState().config.permittedTagTypes;
+              var tagType = res.type;
+
+              if (!permitted.some((e, i, a) => e == tagType)) {
+                dispatch(showWarning('You do not have permission to edit this tag type'));
```

Comment:
Is this the error that is being shown inside Display.js? I wouldn't keep this in the store. This should show the error in ReactApp (but don't use it for this situation)